content/lua/concepts/tables/terms/find/find.md

```diff
@@ -0,0 +1,73 @@
+---
+Title: 'Find()'
+Description: 'Searches for the first occurrence of a specified value within a table.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Tables'
+  - 'Functions'
+CatalogContent:
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
+
+The **find()** is a built in function that searches for a specific value in an a table and returns its index if found or `nil` if the value is not present in the table.
```

Comment:
Find() is not a built in functions in genric lua , Please refer the documentation of the Lua.

Reply:
You are supposed to write the custom function find()